pymeasure/instruments/tektronix/tektronixMsoSeries.py

```diff
@@ -851,7 +851,8 @@ def center_trigger(self):
         """Control the polarity for a pulse width trigger.
         """,
         validator=strict_discrete_set,
-        values=["NEGATIVE", "POSITIVE"],
+        values={"negative": "NEGATIVE", "positive": "POSITIVE"},
+        map_values=True,
```

Comment:
Why not:
```suggestion
        values={False: "NEGATIVE", True: "POSITIVE"},
        map_values=True,
```
maybe also renaming the control to `..positive_polarity`.

Just as a suggestion.

Reply:
Hello Benedikt. For this properties relative to slope and polarity I want to keep it this way. The reason is that Lecroy Scopes has those properties merged i.e slope and polarity. This is not the case for Tektronix scopes. I had an abstraction layer that use Pymeasure to rule any scope and this way I can have negative/positive for both properties whatever the scope used.

Reply:
Ok